### PR TITLE
feat : Added large components

### PR DESCRIPTION
**Pull request by nishasen**
These components are being used at one place and also contain various functionalities.

### CODE REVIEW

**Review by krishanu2911 — APPROVED** [review]
Well written code 
-Well defined function 
-clean and understandable code

File: wiznotes/src/Components/NoteContainer/NoteContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                          value="medium"
+                          label="Medium"
+                          forLabel="medium"
+                          labelClassName="text-black" onChange={(e)=>handleChange(e)} required/>     
+            <CheckboxRadio type="radio"
+                          name="notePriority"
+                          value="low"
+                          label="Low"
+                          forLabel="low"
+                          labelClassName="text-black" onChange={(e)=>handleChange(e)} required/>
```
Comment (sadathshariff) [inline wiznotes/src/Components/NoteContainer/NoteContainer.jsx:95]: you can add checked attribute here

File: wiznotes/src/Components/NoteContainer/NoteContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        </div>
+        {showLabel && <div>
+          <div className="tag-container labels dis-grid">
+                {labels.map(({label, name, value, type, forLabel})=><CheckboxRadio label={label}
+                                                                                    name={name}
+                                                                                    value={value}
+                                                                                    type={type}
+                                                                                    forLabel={forLabel}
+                                                                                    labelClassName={setText}
+                                                                                    onChange={(e)=>handleChange(e)} required/>)}
```
Comment (sadathshariff) [inline wiznotes/src/Components/NoteContainer/NoteContainer.jsx:124]: here as well

File: wiznotes/src/Components/Pages/NotePage.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import './Page.css';
+
+const NotePage = () => {
+  const { noteState } = useNote();
+  const { noteItems } = noteState;
+  return (
+    <div className="page">
+        <SearchTab />
+        <NoteContainer />
+        {/* <MapChip /> */}
```
Comment (sadathshariff) [inline wiznotes/src/Components/Pages/NotePage.jsx:13]: remove commented code

**Review by sadathshariff — APPROVED** [review]
Overall the code looks clean and has good descriptive names,

File: wiznotes/src/Components/NoteContainer/NoteContainer.jsx (hunk 1 above)
Comment (nishasen) [reply wiznotes/src/Components/NoteContainer/NoteContainer.jsx:95]: Okay adding it

**Pull request merged**